sinks/interamap/interamap.py

```
@@ -2,34 +2,77 @@
 import sys
 import os
 
-from PySide6.QtWidgets import QApplication
+from PySide6.QtCore import QTimer
+from PySide6.QtWidgets import QApplication, QMessageBox
+from tileserver import start_tileserver, stop_tileserver
 from src.main_window import MapWindow
 
-from src.config import ONLINE_MODE, MBTILES_PATH
+from config import ONLINE_MODE, MBTILES_PATH
 import subprocess
```

Ruff (0.12.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)